armbian/customize-image-barebone.sh

```diff
@@ -17,9 +17,8 @@ LINUXFAMILY=$2
 BOARD=$3
 BUILD_DESKTOP=$4
 
-PREP_PACKAGE_LIST="avahi-daemon iptables dnsmasq-base"
+ADD_PACKAGE_LIST="avahi-daemon"
```

Comment:
TL;DR variable name consistency. 
Is there a reason why the customize-image-barebone.sh apt package list variable names is changed to ADD_PACKAGE_LIST and customize-image-mainsail.sh and customize-image-fluidd.sh is not changed?

maybe consider the other customize scripts?

Reply:
I think it would be better to move each package list into file with the program that needs it so that we can easily tell which program needs which package instead of having one large list for the whole image. 
For the Barebone image, that list name more reflects what the list is, since there are no top level programs that need it directly. 
It's a bit of work to split the list and find out what packages are needed for what program, but it could be a valuable tool when looking at how much space each top level application needs. 
I can take a look and see if I can fix it at least a little better, but I don't think I want to do the whole splitting job right now.